pkg/i2gw/providers/gce/extensions/input_extensions.go

```diff
@@ -40,6 +44,17 @@ func validateSessionAffinity(beConfig *backendconfigv1.BackendConfig) error {
 	return nil
 }
 
+func validateHealthCheck(beConfig *backendconfigv1.BackendConfig) error {
+	hcType := beConfig.Spec.HealthCheck.Type
+	if hcType == nil {
+		return fmt.Errorf("HealthCheck Protocol type is not specified")
+	}
+	if *hcType != "HTTP" && *hcType != "HTTPS" && *hcType != "HTTP2" {
+		return fmt.Errorf("Protocol %q is not valid, must be one of [HTTP,HTTPS,HTTP2]", *hcType)
+	}
```

Comment:
Nit: This seems like a great thing to move into a higher level variable such as a set. K8s has a nice helper lib for this, and we [already use it in Gateway API](https://github.com/kubernetes-sigs/gateway-api/blob/9f7dd62df47ebdb96bc56626ef22a0ee3b37509c/pkg/features/features.go#L19).

Reply:
Updated.